documentation/Install_PACTA_as_a_first-time_user

```diff
@@ -0,0 +1,137 @@
+# How to install and setup PACTA as a first-time user
+- - -
+## Overview:
+This how-to describes setting up and running a test PACTA analysis for a first time user and how to generate interactive reports.
+
+**Keywords:**
+
+## Before you start:
+### System requirements:
+|           | Recommended   | Minimum               |
+|-----------|---------------|-----------------------|
+| RAM       | 16 GB or more | 8 GB                  |
+| Windows   | 64-bit        | 32-bit                |
+| Mac OS    |               |                       |
```

Comment:
@AlexAxthelm do you have info on system requirements?

Reply:
@meggpie same across all platforms.